app/src/main/res/values/strings.xml

```diff
@@ -328,4 +328,9 @@
     <string name="room_state_too_full">too full</string>
     <string name="room_state_not_retrievable">not retrievable</string>
 
+    <!-- Search filters -->
+    <string name="search_filter_is_favorite">Is favored</string>
+    <string name="search_filter_has_alarm">Has alarm</string>
+    <string name="search_filter_not_recorded">Not recorded</string>
+    <string name="search_filter_within_speaker_names">Within speaker names</string>
```

Comment:
Let's add the German translation - it is not part of the translation platform. Here are my proposals:

- Ist favorisiert
- Hat Alarm
- Nicht aufgezeichnet
- In Namen von Sprecher*innen

Please also test how the screen behaves with TalkBack. We might need to add content descriptions, at least for the last one: "In Namen von Sprecher innen".